apps/src/componentLibrary/typography/typography.module.scss

```diff
@@ -0,0 +1,173 @@
+@import "color", "font";
+
+/* This file is a adopted copy of shared/css/typography.scss
+  updating styles in this file will require a manual sync with the that file
+*/
+
+// TODO: Once we move Typography to DSCO_ - make sure we delete apps/fonts/ folder,
+//  since that folder is only needed for scss modules
+@font-face {
+  font-family: "Barlow Semi Condensed Semibold";
+  font-style: normal;
+  font-weight: 600;
+  src:
+    url("/fonts/barlowSemiCondensed/BarlowSemiCondensed-SemiBold.ttf") format("truetype"),
+    local("?");
+}
+
+@font-face {
+  font-family: "Barlow Semi Condensed Medium";
+  font-style: normal;
+  font-weight: 500;
+  src: url("/fonts/barlowSemiCondensed/BarlowSemiCondensed-Medium.ttf") format("truetype"), local("?");
+}
+
+// This stylesheet can be imported to maintain typographic
+// consistency across Dashboard and Pegasus sites.
+
+// Includes heading, paragraph, and caption styles.
+
+// We're using mixins to style semantic tags and also creating
+// identical classes for potential use on other elements if needed.
+
+// ----------------------------------------
+
+// Heading styles
+// Heading mixins are used to maintain accessible heading hierarchy
+// when design calls for a different style (semantic h3, h5 styling).
+
+// Use .heading-[xx] classes on single elements in the markup.
+// This is an h3 heading that will look like an h5.
+// - example: h3.heading-sm
+
+// Use @include [mixin-name] in stylesheets for widespread use.
+
+/* — example: all h3s in a ul need to look like h5s
+    ul.icon-list {
+      h3 { @include heading-sm }
+    }
+*/
+
+@mixin heading-common {
+  color: $neutral-dark;
+  line-height: 1.2;
+  margin: 0 0 0.5em 0;
+}
+
+@mixin heading-xxl {
```

Comment:
Definitely out of scope of this PR, but if there's a way to export types from CSS modules (i think there is?), it would be great to enforce some type checking such that all the styles defined here are referenced in `index.ts`

Reply:
Good point! I feel like there is, but not sure yet. Added a ticket to Jira SL backlog to investigate and not forget about it, [SL-795](https://codedotorg.atlassian.net/browse/SL-795?atlOrigin=eyJpIjoiZmI5Yzc4MDBkMmZmNDViN2EyOTljNTQ3YjMwOGFlYjEiLCJwIjoiaiJ9)